content.js

```diff
@@ -1,5 +1,15 @@
-var webhookUrl
-chrome.storage.sync.get('webhookUrl', (data) => (webhookUrl = data.webhookUrl))
+let webhookUrl
+
+chrome.storage.sync.get('diceBot', ({ diceBot }) => {
+    const { characters } = diceBot
+
+    webhookUrl = characters.reduce((found, character) => {
+        if (character.ddbUrl === window.location.href) {
+            return character.discordUrl
+        }
+        return found
+    }, '')
+})
```

Comment:
Changed the data structure in Chrome storage so this had to change, too. Check the `initialState` variable in the options reducer for what it will look like going forward.